test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -104,17 +105,17 @@ class CloudFoundryDeployTest extends BasePiperTest {
                 cloudFoundryDeploy: []
             ]
         ]
-
+        nullScript.commonPipelineEnvironment.setBuildTool('mta')
```

Comment:
Why do we need to change this?

Reply:
To test whether the automatic inference of the deployTool based on the buildTool works.

Reply:
ah, understood.